apps/examples/testcase/ta_tc/ble_manager/itc/itc_blemanager_main.c

```diff
@@ -0,0 +1,865 @@
+{
+	printf("client connected callback received\n");
+	ITC_FUNC_SIGNAL;
+	return;
+}
+
+static void ble_operation_notification_cb(ble_client_ctx *ctx, ble_attr_handle attr_handle, ble_data *read_result)
+{
+	printf("notification callback received\n");
+	ITC_FUNC_SIGNAL;
+	return;
+}
+
+static void ble_server_connected_cb(ble_conn_handle con_handle, ble_server_connection_type_e conn_type, uint8_t mac[BLE_BD_ADDR_MAX_LEN])
+{
+	printf("server connect callback received");
+	return;
+}
+
+static ble_server_gatt_t gatt_profile[] = {
+	{
+		.type = BLE_SERVER_GATT_SERVICE,
+		.uuid = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x01, 0x01, 0x01},
+		.uuid_length = 16,
+		.attr_handle = 0x006a,
+	},
+
+	{
+		.type = BLE_SERVER_GATT_CHARACT,
+		.uuid = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x01, 0x01, 0x02},
+		.uuid_length = 16,
+		.property = BLE_ATTR_PROP_RWN | BLE_ATTR_PROP_WRITE_NO_RSP,
+		.permission = BLE_ATTR_PERM_R_PERMIT | BLE_ATTR_PERM_W_PERMIT,
+		.attr_handle = 0x006b,
+		.arg = "char_a_1"
+	},
+
+	{
+		.type = BLE_SERVER_GATT_DESC,
+		.uuid = {0x02, 0x29},
+		.uuid_length = 2,
+		.permission = BLE_ATTR_PERM_R_PERMIT | BLE_ATTR_PERM_W_PERMIT,
+		.attr_handle = 0x006c,
+		.arg = "desc_b_1",
+	},
+};
+
+static uint8_t g_adv_raw[] = {
+	0x02, 0x01, 0x05, 0x03, 0x19, 0x80, 0x01, 0x05, 0x03, 0x12, 0x18, 0x0f, 0x18
+};
+static uint8_t g_adv_resp[] = {
+	0x11, 0x09, 'T', 'I', 'Z', 'E', 'N', 'R', 'T', ' ', 'T', 'E', 'S', 'T', '(', '0', '2', ')',
+};
+
+static ble_scan_callback_list scan_config = {
+	ble_scan_state_changed_cb,
+	ble_device_scanned_cb,
+};
+
+static ble_client_callback_list client_config = {
+	ble_device_disconnected_cb,
+	ble_device_connected_cb,
+	ble_operation_notification_cb,
+};
+
+static ble_server_init_config server_config = {
+	ble_server_connected_cb,
+	true,
+	gatt_profile,
+	sizeof(gatt_profile) / sizeof(ble_server_gatt_t)
+};
+
+static void set_scan_filter(ble_scan_filter *filter, uint8_t *raw_data, uint8_t len, bool whitelist_enable, uint32_t scan_duration)
+{
+	memset(filter, 0, sizeof(ble_scan_filter));
+	if (raw_data != NULL && len > 0) {
+		memcpy(filter->raw_data, raw_data, len);
+		filter->raw_data_length = len;
+	}
+
+	filter->scan_duration = scan_duration;
+	filter->whitelist_enable = whitelist_enable;
+}
+
+static double diff_time(struct timeval *x, struct timeval *y)
+{
+	double x_ms;
+	double y_ms;
+	double diff;
+	x_ms = (double)x->tv_sec * 1000000 + (double)x->tv_usec;
+	y_ms = (double)y->tv_sec * 1000000 + (double)y->tv_usec;
+	diff = (double)y_ms - (double)x_ms;
+	return diff;
+}
+
+/*********************************************************************
+ * Test Cases
+ *********************************************************************/
+
+/**
+ * @testcase         itc_blemanager_init_deinit_p
+ * @brief            to check initialize and de-initalization of ble manager
```

Comment:
please change "initialize" to "initialization" to main consistency.

Reply:
Sure, done